src/services/user.rs

```diff
@@ -178,3 +187,121 @@ pub fn auth_and_delete_integration<T: user::UserModel + integration::Integration
     let _ = jwt::verify_token(Utc::now().naive_utc(), token, jwt_secret)?;
     Ok(database.delete_integration(&id)?)
 }
+
+#[cfg(test)]
+mod user_tests {
+
+    use super::*;
+
+    use crate::entities::user;
+
+    struct T {}
+
+    impl user::UserModel for T {
+        fn create_user(&self, user: user::NewUser) -> user::Result<user::User> {
+            Ok(user::User {
+                id: Uuid::from_u128(160141200314647599499076565412518613020),
+                name: user.name,
+                username: user.username,
+                register_date: None,
+                email: user.email,
+                last_code_gen_request: None,
+                login_code: None,
+                is_registered: true,
+                payday: None,
+            })
+        }
+        fn delete_user(&self, id: &Uuid) -> user::Result<user::User> {
+            Ok(user::User {
+                id: Uuid::from_u128(160141200314647599499076565412518613020),
+                name: "Usuário 1".to_string(),
+                username: "usuario1".to_string(),
+                register_date: None,
+                email: "usuario1@gmail.com".to_string(),
+                last_code_gen_request: None,
+                login_code: None,
+                is_registered: true,
+                payday: None,
+            })
+        }
```

Comment:
Why are you generating an uuid if you passed one as a parameter?

Reply:
fixed